manifests/service.pp

```diff
@@ -43,6 +46,7 @@
   $service_name        = $title,
   $tags                = [],
   $token               = undef,
+  $meta                = {},
```

Comment:
Can you please add a datatype to the new parameter? We would like to have them for all new parameters.